docs/sql-performance-tuning.md

```diff
@@ -286,6 +286,27 @@ This feature coalesces the post shuffle partitions based on the map output stati
    </tr>
  </table>
  
+### Spliting skewed shuffle partitions
+ <table class="table">
+   <tr><th>Property Name</th><th>Default</th><th>Meaning</th><th>Since Version</th></tr>
+   <tr>
+     <td><code>spark.sql.adaptive.optimizeSkewsInRebalancePartitions.enabled</code></td>
+     <td>true</td>
+     <td>
+       When true and <code>spark.sql.adaptive.enabled</code> is true, Spark will optimize the skewed shuffle partitions in RebalancePartitions and split them to smaller ones according to the target size (specified by <code>spark.sql.adaptive.advisoryPartitionSizeInBytes</code>), to avoid data skew.
+     </td>
+     <td>3.2.0</td>
+   </tr>
+   <tr>
+     <td><code>spark.sql.adaptive.rebalancePartitionsSmallPartitionFactor</code></td>
```

Comment:
This is a non-internal configuration which has been defined and used for a while.

https://github.com/apache/spark/blob/786594734bd79017ebd42eb117b62958afad07bb/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L792-L799